modules/plugin/epsg-hsql/src/main/java/org/geotools/referencing/factory/epsg/hsql/ThreadedHsqlEpsgFactory.java

```diff
@@ -276,6 +277,8 @@ protected AbstractAuthorityFactory createBackingStore(final Hints hints) throws
                     byte[] buf = new byte[1024];
                     int read = 0;
                     while ((ze = zin.getNextEntry()) != null) {
+                        Utilities.assertNotZipSlipVulnarable(
+                                new File(directory, ze.getName()), directory.toPath());
```

Comment:
We expect this change has introduced a problem for GeoTools 20.1; GeoServer is failing to find the EPSG authority when running as a windows service.

Reply:
If the original author is not coming forward, I'd support just removing this line, the zip file is in the jar and we generated it, so it should not contain any zip-slip attack.

Reply:
Here is the actual failure:

```
Bad zip entry: ZipSlip extracting C:\Program Files (x86)\GeoServer 2.15.1\.\work\GeoTools\Databases\HSQL\v8.6.0.1\EPSG.data to C:\Program Files (x86)\GeoServer 2.15.1\.\work\GeoTools\Databases\HSQL\v8.6.0.1
```